src/utils/db.ts

```diff
@@ -0,0 +1,48 @@
+import { Level } from "level";
+
+export type DBLevel = Level<string, string>;
+
+export default class DBService {
+  protected db: DBLevel;
+
+  private static _instance: DBService = new DBService();
+
+  protected constructor() {
+    if (DBService._instance) {
+      throw new Error(
+        "Error: Instantiation failed: Use DBService.getInstance() instead of new."
+      );
+    }
+    DBService._instance = this;
+
+    if (process.env.NODE_IS_TEST === "true") {
```

Comment:
why not using NODE_ENV

also, i don't think the this class should be aware of env vars, it should receive all params in the constructor

Reply:
Paths are passed to the constructor, ready for stitching in with env aware logic, however will defer this to #60.